Cargo.toml

```diff
@@ -15,6 +15,7 @@ categories = ["development-tools", "development-tools::cargo-plugins", "developm
 license = "Apache-2.0"
 
 [dependencies]
+num_cpus = "1"
```

Comment:
Can we use `std::thread::available_parallelism` instead of adding the dep?

Reply:
Neat! TIL about that API